bin/api-docs/gen-theme-reference.mjs

```diff
@@ -9,6 +9,7 @@
  */
 import fs from 'node:fs/promises';
 import $RefParser from '@apidevtools/json-schema-ref-parser';
+import { fileURLToPath } from 'url'
```

Comment:
[Fun, I'd never seen this function:](https://nodejs.org/api/url.html#urlfileurltopathurl-options)

> This function ensures the correct decodings of percent-encoded characters as well as ensuring a cross-platform valid absolute path string.

I really like the `node:` prefixed style for builtin modules:

```suggestion
import { fileURLToPath } from 'node:url'
```